tests/.coveragerc

```diff
@@ -1,8 +1,5 @@
 [run]
 source = sorl
-exclude_lines =
-    pragma: no cover
-    if __name__ == .__main__.:
```

Comment:
I don't know. If I leave those lines in, when I run tox I see:

```
$ tox -e py27-django18-vipsthumbnail
py27-django18-vipsthumbnail create: /home/john/GIT/sorl-thumbnail/.tox/py27-django18-vipsthumbnail
py27-django18-vipsthumbnail installdeps: pytest, pytest-cov, pytest-django, Pillow, Django>=1.8,<1.9
py27-django18-vipsthumbnail runtests: PYTHONHASHSEED='2779594240'
py27-django18-vipsthumbnail runtests: commands[0] | py.test -rw --cov-config .coveragerc --cov sorl
Traceback (most recent call last):
  File "../.tox/py27-django18-vipsthumbnail/bin/py.test", line 11, in <module>
    sys.exit(main())
... snip ...
  File "/home/john/GIT/sorl-thumbnail/.tox/py27-django18-vipsthumbnail/local/lib/python2.7/site-packages/coverage/config.py", line 241, in from_file
    section, unknown, filename
coverage.misc.CoverageException: Unrecognized option '[run] exclude_lines=' in config file .coveragerc
ERROR: InvocationError: '/home/john/GIT/sorl-thumbnail/.tox/py27-django18-vipsthumbnail/bin/py.test -rw --cov-config .coveragerc --cov sorl'
___________________________________ summary ____________________________________
ERROR:   py27-django18-vipsthumbnail: commands failed
```

This is on ubuntu 15.04 with tox 1.8.0. I'm sure there's a better fix, but deleting those lines got tox working for me.